x/validator-vesting/client/cli/query.go

```diff
@@ -5,212 +5,237 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
-	"github.com/cosmos/cosmos-sdk/codec"
 
 	"github.com/kava-labs/kava/x/validator-vesting/types"
 )
 
 // GetQueryCmd returns the cli query commands for the kavadist module
-func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+func GetQueryCmd() *cobra.Command {
 	valVestingQueryCmd := &cobra.Command{
 		Use:   types.QueryPath,
 		Short: "Querying commands for the validator vesting module",
 	}
 
-	valVestingQueryCmd.AddCommand(flags.GetCommands(
-		queryCirculatingSupply(queryRoute, cdc),
-		queryTotalSupply(queryRoute, cdc),
-		queryCirculatingSupplyHARD(queryRoute, cdc),
-		queryCirculatingSupplyUSDX(queryRoute, cdc),
-		queryCirculatingSupplySWP(queryRoute, cdc),
-		queryTotalSupplyHARD(queryRoute, cdc),
-		queryTotalSupplyUSDX(queryRoute, cdc),
-	)...)
+	cmds := []*cobra.Command{
+		queryCirculatingSupply(),
+		queryTotalSupply(),
+		queryCirculatingSupplyHARD(),
+		queryCirculatingSupplyUSDX(),
+		queryCirculatingSupplySWP(),
+		queryTotalSupplyHARD(),
+		queryTotalSupplyUSDX(),
+	}
+
+	for _, cmd := range cmds {
+		flags.AddQueryFlagsToCmd(cmd)
+	}
 
+	valVestingQueryCmd.AddCommand(cmds...)
 	return valVestingQueryCmd
 
 }
 
-func queryCirculatingSupply(queryRoute string, cdc *codec.Codec) *cobra.Command {
+func queryCirculatingSupply() *cobra.Command {
 	return &cobra.Command{
 		Use:   "circulating-supply",
 		Short: "Get circulating supply",
 		Long:  "Get the current circulating supply of kava tokens",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			cliCtx, err := client.GetClientTxContext(cmd)
```

Comment:
```suggestion
			cliCtx, err := client.GetClientQueryContext(cmd)
```

I switched these up a lot before too, same for the other query commands